vb-fe/src/components/sheet/Sheet.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, FunctionComponent } from "react";
+
+import { useDispatch } from "react-redux";
+import { Box, IconButton } from "@chakra-ui/react";
+import { IoIosArrowUp, IoIosArrowDown } from "react-icons/io";
+import { MIN_SHEET_HEIGHT, MAX_SHEET_HEIGHT } from "util/Variables";
+import { SystemConstants } from "_store/system/SystemTypes";
+
+export const Sheet: FunctionComponent<any> = (): JSX.Element => {
```

Comment:
I don't think we need the `<any>` here, right?  It doesn't take any args.  Also, is the return for functional components inferred?  We might not need to type it

Reply:
oh it will take in some props later, so its just a placeholder rn.

and i think you need to add return types to elements, otherwise the linter complains lol